src/app.js

```diff
@@ -141,13 +148,36 @@ class Messenger extends EventEmitter {
       pageId: config.get('facebook.pageId')
     }));
 
-    this.app.get('/', (req, res) => {
-      res.send('👍');
-    });
+    this.app.post('/pause', bodyParser.json(), (req, res) => {
+      const { userId, paused } = req.body;
+      if (!userId && paused === undefined) {
+        res.sendStatus(400);
+        return;
+      }
 
-    this.app.get('/ping', (req, res) => {
-      res.send('pong');
+      const cacheKey = this.getCacheKey(userId);
+      this.cache.get(cacheKey)
+        .then((session/*: ?Session */)/*: Promise<Session> */ => {
+          if (!session) {
+            throw new Error("Can't pause unknown user");
+          }
+          session.paused = paused ? Date.now() : false;
```

Comment:
an alternative would be to just remove the key to reduce the size of the request to Redis, but I guess I'll just leave this as is. If we were able to do multigets [`MGET`](https://redis.io/commands/mget), then payload size would actually start getting important.